cli/src/action/circuit/mod.rs

```diff
@@ -28,7 +28,9 @@ impl Action for CircuitCreateAction {
     fn run<'a>(&mut self, arg_matches: Option<&ArgMatches<'a>>) -> Result<(), CliError> {
         let args = arg_matches.ok_or_else(|| CliError::RequiresArgs)?;
         let url = args.value_of("url").unwrap_or("http://localhost:8085");
-        let key = args.value_of("private_key_file").unwrap_or("splinter");
+        let key = args
+            .value_of("private_key_file")
+            .unwrap_or("./splinter.priv");
```

Comment:
This might be a security issue. Instead of defaulting to the cwd, hard-code the /etc directory location.

Reply:
But this is a user key, so this default is based on the default generated when using keygen on a user basis, not the admin private key.

Reply:
Ok, I did miss that - use a path int he user's home directory similar to how it works with the sawtooth tools.

Reply:
I will fix that in both places

Reply:
I think this should be addressed in a separate PR, as it requires the `keygen` subcommand to be modified.  Looking at it again, it looks to be under `splinter admin keygen` which doesn't seem right for a user key.  We should consider moving this to `splinter keygen`, with the user's home directory as the default target, as in the Sawtooth tools.

Reply:
Agreed.